### PR TITLE
Elastic practice

### PR DESCRIPTION
**Pull request by carrierce**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,17 +4,21 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Test Elasticlunr</title>
+    <link rel="stylesheet" href="main.css">
+    <title>Coder Bunker Member Search</title>
```
Comment (rngadam) [inline index.html]: Branding is Coderbunker

File: index.html [hunk 2]
```diff
@@ -4,17 +4,21 @@
...
-    <div>
-        <input id="search" name="search" />
-        <button class="search-btn">Search</button>
-    </div>
-    
-    <p class="final-text"></p>
-
+    <div class="search-section">
+        <div class="search-area">
+            <h2>#CODERBUNKER MEMBER SEARCH</h2>
```
Comment (rngadam) [inline index.html]: Coderbunker Members Search

File: js/index.js [hunk 3]
```diff
@@ -2,33 +2,71 @@ const inputArea = document.getElementById("search");
...
+        'profile': '',
+    };
+    let allUsersArray = [];
+
     const searchStr = inputArea.value;
-    const matchArr = index.search(searchStr);  // here's the result
-    console.log(matchArr);
-    finalTextArea.innerText = JSON.stringify(matchArr);  // too lazy to style the result ;P
+    const matchArr = index.search(searchStr);
+    for ( let g = 0; g < matchArr.length; g++) {
```
Comment (ortonomy) [inline js/index.js]:
```
const results = matchArr.map( match => {
  return {
    name: match.doc.fullname,
    profile: match.doc.profileUrl
  }
})

console.log(results) // [{ name: ..., profile: ...}, ...]
```

File: js/index.js [hunk 4]
```diff
@@ -2,33 +2,71 @@ const inputArea = document.getElementById("search");
...
 
-// you can grab the data from frontend, but I'm too lazy to do that
-var doc1 = {
-    "id": 1,
-    "title": "Oracle released its latest database Oracle 12g",
-    "body": "Yesterday Oracle has released its new database Oracle 12g, this would make more money for this company and lead to a nice profit report of annual year."
-}
+let membersArray = [];
+let request = new XMLHttpRequest();
+let APIEndPoint = "";
```
[the remark this answers is not in the document]
Comment (ortonomy) [reply js/index.js]: This should be more advice, -- this can be a future improvement.

**Pull request merged**